jekyll/_cci2/workflows.md

```diff
@@ -56,23 +56,24 @@ _For a full specification of the_ `workflows` _key, see the [Workflows]({{ site.
 To run a set of concurrent jobs, add a new `workflows:` section to the end of your existing `.circleci/config.yml` file with the version and a unique name for the workflow. The following sample `.circleci/config.yml` file shows the default workflow orchestration with two concurrent jobs. It is defined by using the `workflows:` key named `build_and_test` and by nesting the `jobs:` key with a list of job names. The jobs have no dependencies defined, therefore they will run concurrently.
 
 ```yaml
-jobs:
-  build:
-    docker:
-      - image: circleci/<language>:<version TAG>
-    steps:
+version: 2.1
+jobs: 
+  build: 
+    docker: 
+        image: "circleci/<language>:<version TAG>"
+    steps: 
       - checkout
-      - run: <command>
-  test:
-    docker:
-      - image: circleci/<language>:<version TAG>
-    steps:
+        run: <command>
+  test: 
+    docker: 
+        image: "circleci/<language>:<version TAG>"
+    steps: 
       - checkout
-      - run: <command>
-workflows:
-  version: 2
-  build_and_test:
-    jobs:
+        run: <command>
+version: 2.1
```

Comment:
We only need one version at the top for the config. In `version: 2` config we also needed to specify `version: 2` for workflows but this isn't required for `2.1`.